src/resources/lessonFrontmatter.js

```diff
@@ -58,14 +58,14 @@ const getData = memoize(
     for (const key of lessonFrontmatterContext.keys()) {
       const [/* ignore */, course, lesson, file] = key.match(/^[.][/]([^/]+)[/]([^/]+)[/]([^.]+)[.]md$/);
       const {
-        language, title = '', level = 0, author = '', translator = '', external = ''
+        language, title = '', level = 0, author = '', translator = '', external = '', license = ''
       } = lessonFrontmatterContext(key);
       if (!title) { console.warn('WARNING: The lesson', key, 'did not specify title.'); }
       if (language) {
         const isReadmeKey = file.startsWith('README') ? 1 : 0;
         const path = `/${course}/${lesson}/${file}`; // TODO: Add publicpath?
         const pdfPath = `${path}.pdf`;
-        const lessonData = {title, level, author, translator, external, path, pdfPath, key};
+        const lessonData = {title, level, author, translator, external, path, pdfPath, key, license};
```

Comment:
@tjesi foreslår at lisensen skal ligge i `lesson.yml`, altså at den hentes fra `resources/lessons.js`. Men se kommentaren min, for kanskje det faktisk er bedre slik du har gjort det @Skagevang, med mulighet for å ha lisens i frontmatter for hver oversettelse?